<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>741</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Programs</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="Numero_Sem">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>181</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID_Planning">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QDateEdit" name="Date_Planning">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Heure:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NB_reservation">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>231</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>NB_reservation:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>181</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero_Sem:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Nom_Coach">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter2">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>300</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="Heure">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>279</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom_Coach:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Planning:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_Planing:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>481</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Afficher2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Refrech</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QPushButton" name="Modifier2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Coach</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID_Coach_modf2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="ID_Coach_Sup2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>69</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Supprimer2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Coach</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Coach</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="Specialite">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Prenom">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="Date_Naiss">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Specialité:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Nom">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_Naiss :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter1">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>310</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Coach</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID_Coach">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>471</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Afficher1">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Refrech</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID_Coach_modf1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Coach</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Modifier1">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>190</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="Supprimer1">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell Extra Bold</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID_Coach_sup1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>79</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Coach</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
